docs/markdown/Builtin-options.md

```diff
@@ -92,8 +92,9 @@ machine](#specifying-options-per-machine) section for details.
 | unity_size {>=2}                       | 4             | Unity file block size                                          | no             | no                |
 | warning_level {0, 1, 2, 3, everything} | 1             | Set the warning level. From 0 = none to everything = highest   | no             | yes               |
 | werror                                 | false         | Treat warnings as errors                                       | no             | yes               |
-| wrap_mode {default, nofallback,<br>nodownload, forcefallback, nopromote} | default | Wrap mode to use                 | no             | no                |
+| wrap_mode {default, nofallback,<br>nodownload, forcefallback, nopromote} | default | Wrap mode to use                   | no             | no                |
 | force_fallback_for                     | []            | Force fallback for those dependencies                          | no             | no                |
+| vsenv                                  | false         | Activate Visual Studio environment                             | no             | no                |
```

Comment:
Since 1.1.0?

Reply:
Hm, `-Dvsenv` is new since 1.1.0 but `--vsenv` was already there. Not sure if/how to document that.

Reply:
Something like: `*Since 1.1.0*. Was a command line argument before.`?

Reply:
Added into the doc all the details we had in release notes snippet of original PR: https://github.com/mesonbuild/meson/commit/928078982c8643bffd95a8da06a1b4494fe87e2b.